lib/mongo/server/monitor/connection.rb

```diff
@@ -131,7 +133,7 @@ def initialize(address, options = {})
         #
         # @since 2.0.0
         def timeout
-          @timeout ||= options[:connect_timeout] || CONNECT_TIMEOUT
+          @timeout ||= options[:connect_timeout]
```

Comment:
Yes indeed, If it's used in multiple places I would probably move it up to the Mongo module or Configuration one, but that's a matter of taste and conventions you can do what you want ;)